package.json

```diff
@@ -40,15 +42,12 @@
     "@types/jest": "29.0.0",
     "@types/node": "18.7.14",
     "@types/supertest": "2.0.12",
-    "@typescript-eslint/eslint-plugin": "5.36.1",
-    "@typescript-eslint/parser": "5.36.1",
     "cross-env": "7.0.3",
     "cspell": "6.8.1",
     "eslint": "8.23.0",
-    "eslint-config-prettier": "8.5.0",
-    "eslint-plugin-prettier": "4.2.1",
     "husky": "8.0.0",
     "jest": "28.0.0",
+    "lint-staged": "13.0.3",
```

Comment:
Don't forget to add all required dependencies; please remove lint-staged from your global environment to avoid these kind of problems.